website/www/site/layouts/case-studies/list.html

```diff
@@ -1,112 +1,72 @@
-        without category
-      </button>
-      {{ end }}
-    </div>
-
-    <div class="use_cases">
-        {{ range $pages.ByPublishDate.Reverse }}
-          <a href="{{ .RelPermalink }}">
-            <div class="use-card remove" data-categories='{{ with .Params.categories }}{{ if eq ( printf "%T" . ) "string" }}{{ . | replaceRE "[.]" "_" | urlize }}{{ else if eq ( printf "%T" . ) "[]string" }}{{ range . }}{{ . | replaceRE "[.]" "_" | urlize }} {{end}}{{end}}{{else}}no-author{{end}}'
-            >
-              <div class="use-icon">
-                <img src="{{.Params.icon}}"></i>
-              </div>
-              <div class="use-body">
-                <div>
-                  <p class="use-category">
-                    {{ delimit .Params.categories ", " " & " }}
-                  </p>
-                  <h5 class="use-title">{{ .Title }}</h5>
-                  <p class="use-summary">{{ .Summary | plainify | chomp }}</p>
-                </div>
-                <a class="read-link" href="{{ .RelPermalink }}">{{ T "case-studies-read" }}</a>
-              </div>
+  <h1>Case Studies</h1>
+  <p class="case-study-about">Apache Beam powers many of today’s leading projects, industry-specific use cases, and
+    startups</p>
+  {{ $pages := .Pages }}
+  <div>
+    {{ range where $pages "Params.category" "study" }}
+    <div class="case-study-card">
+      <div class="case-study-card-img">
+        <img src="{{.Params.icon}}"></i>
+      </div>
+      <h3 class="case-study-card-title">{{ .Params.cardTitle }}</h3>
+      <p class="case-study-card-description">{{ .Params.cardDescription }}</p>
+      <div class="case-study-quote-block">
+        <div class="case-study-quote-author">
+          <div class="case-study-quote-author-img">
+            <img src="{{ .Params.authorImg }}">
+          </div>
+          <div class="case-study-quote-author-info">
+            <div class="case-study-quote-author-name">
+              {{ .Params.authorName }}
             </div>
-          </a>
-        {{ end }}
+            <div class="case-study-quote-author-position">
+              {{ .Params.authorPosition }}
+            </div>
+          </div>
+        </div>
+      </div>
+      <a class="case-study-card-learn-more" href="{{ .RelPermalink }}">
+        <span>{{ T "case-studies-learn-more" }}</span>
+        <img src="/images/arrow-right.svg" alt="Go to the case study">
+      </a>
     </div>
-    <div class="category-buttons category-buttons_center">
-        <button id="load-button" class="load-button">{{ T "blog-loadmore" }}</button>
+    {{ end }}
+  </div>
+  <div class="case-study-row-button-container">
+    <a href="" class="case-study-primary-button">Share your story</a>
+  </div>
+  <h2 class="case-study-h2">Also used by</h2>
+  <div class="case-study-list">
+    {{ range where $pages "Params.category" "ne" "study" }}
```

Comment:
Was it sorted by $pages.ByPublishDate.Reverse? Perhaps we should keep this sorting

Reply:
Updated, thanks